e2e/README.md

```diff
@@ -23,12 +23,24 @@
 - Use environment variables which described in the **```'TestConstants.ts'```** file
 
 ## Docker launch
+
 - open terminal and go to the "e2e" directory
```

Comment:
@Ohrimenko1988: we might not need to go to the "e2e" directory if we don't mount it to docker container. WDYT?

Reply:
Totally disagree, for launch the "npm run test-docker" command you exactly should be in the root of e2e project or bash will not recognize "npm" command

Reply:
Thanks for clarification.